packages/calypso-products/src/translations.tsx

```diff
@@ -1239,6 +1244,6 @@ export const useJetpackStorageAmountTextByProductSlug = (): ( (
 				[ PLAN_JETPACK_SECURITY_T2_MONTHLY ]: ONE_TERABYTE,
 				[ PLAN_JETPACK_SECURITY_T2_YEARLY ]: ONE_TERABYTE,
 			}[ productSlug ] ),
-		[ TEN_GIGABYTES, HUNDRED_GIGABYTES, ONE_TERABYTE, THREE_TERABYTE, FIVE_TERABYTE ]
+		[ TEN_GIGABYTES, ONE_TERABYTE, HUNDRED_GIGABYTES, THREE_TERABYTE, FIVE_TERABYTE ]
```

Comment:
I'm not sure if this was intentionally changed, but the previous order looks more right (10gb, 100gb, 1tb...)